x-pack/plugin/src/test/resources/rest-api-spec/test/ml/inference_crud.yml

```diff
@@ -851,3 +827,89 @@ setup:
         model_id: "a-regression-model-1"
         include_model_definition: true
         decompress_definition: false
+---
+"Test put model model aliases":
+
+  - do:
+      ml.put_trained_model_alias:
+        model_alias: "regression-model"
+        model_id: "a-regression-model-1"
+  - do:
+      ml.get_trained_models:
+        model_id: "regression-model,a-classification-model"
```

Comment:
Pls update the GET trained model docs to say the parameter can be an alias or model id. See ml-shared.asciidoc#1148

Reply:
yep!